### PR TITLE
changing the function a bit

### PR DESCRIPTION
**Pull request by Mademoisellefou**
I think it would be better if the methods returned more filtered strings,using regular expressions we can filter those questions and answers.

### CODE REVIEW

File: scripts/question_utils.py [hunk 1]
```diff
@@ -5,63 +5,54 @@
 import pathlib
 from random import choice
 from typing import List
-
-p = pathlib.Path(__file__).parent.parent.joinpath('README.md')
+import os
+'''
+devops-exercises//README.md
+'''
+p = os.getcwd()+'/README.md'
```
Comment (bregman-arie) [inline scripts/question_utils.py:12]: Don't you assume here that the script runs from the root of the repo? What happens if you don't run it from the root of the directory?

Comment (Mademoisellefou) [reply scripts/question_utils.py]: you're right, i'll fix it,thanks.

**Comment by bregman-arie** [comment]
@Mademoisellefou thank you for the change, but can you please do PEP8 check? code styling right now isn't 100% right. One way to test if code adheres to PEP8 is by running flake8 (if you are inside the scripts directory then "flake8 ."). Thanks

File: scripts/question_utils.py [hunk 2]
```diff
@@ -5,79 +5,85 @@
...
 >> 123
-
+ # noqa: E501
 """
 
 if __name__ == "__main__":
     import doctest
+
     doctest.testmod()
+    # print(get_question_list(get_file_list()))
```
Comment (bregman-arie) [inline scripts/question_utils.py:86]: Do we need these comments?

**Comment by bregman-arie** [comment]
Thank you for making additional changes. Left a question inline.

**Comment by Mademoisellefou** [comment]
if we disconnect the first may not be necessary, you're right.If we
discomnet the 2, 3 and 4 we will receive a question.

**Pull request merged**